docs/hiring-guide/community-spotlight/interview-liz-harris.rst

```diff
@@ -0,0 +1,141 @@
+
+
+What then led you to technical writing?
+
+    Well, I loved teaching. I was a teacher at a private school in San Francisco, at a French school. 
+    And loved it, but it was the late nineties. There was a tech boom going on in the Bay Area, and I 
+    wasn't participating in it. There was a lot of money being made, and it was so easy to get a job. 
+    The hiring was ferocious. There were people who I knew from growing up in the area working in tech, 
+    who I knew were no smarter than or no more hardworking than me, and they had these jobs that were better 
+    paid than a teacher's job, but also interesting and appealing. The only reason I had heard of the 
+    profession of a technical writer was from a stepmother who was in the family for a while. She was a 
+    technical writer. She actually was a poet and an English teacher, and then she became a technical 
+    writer in the eighties. So I think it was 1998. I had tried to stay in teaching, but I didn't have a 
+    teaching credential. The reason I was looking for a teaching job is I really like teaching. But my 
+    particular school where I was working was a dysfunctional mess. So after three years in a row of not 
+    finding a different teaching job, I thought forget it. I'm just going to do something else. And so I 
+    took a huge career leap. No, it's not that much of a career leap when you're 27 and don't have a mortgage 
+    and don't have children. There wasn't that much at stake. But my husband was in graduate school, so my 
+    teaching job was paying the rent. It was a bit of a leap in that way. So I put out an application or two 
+    and got a job.
+
+
+Liz's first job as a tech writer involved creating documentation for startup called Prototype that 
+produced software for managing fleets of vehicles, like taxis or limousines. Afterwards she spent 8 
+years as an editor and writer for Wind River, a company that produced microprocessors. Afterwards she 
+moved on to Aspera where she works now.
+
+
+Experience as a technical writer
+----------------------------------------------------
+
+How did you differentiate yourself during your initial job search?
+
+    I had to have a portfolio of sorts, and this is long enough ago that it was still pretty usual for 
+    people to show up to an interview with paper examples of their portfolio. So I thought about what I 
+    could show. I chose a couple of things that I had produced as a teacher, like a handout that explains 
+    some grammatical or punctuation concepts, or it might have been an exam or a grading rubric. They were 
+    artifacts from my teaching life. But they did show that I could write cogent copy, that I had careful a 
+    touch of detail. But but I think that I really had it in my favor that it was 1998. The job market was 
+    crazy.
+
+Liz stayed at her first technical writing job for around two years and then moved on to Wind River, a 
+company that specialized in embedded systems.
```

Comment:
I don't quite follow this. Is it a summary, then continues on with what she said?

Reply:
Yeah, more or less. For the sake of conciseness, I cut that portion of the interview, and this is my attempt at bridging between them. Do you think I should format the text differently, or maybe move it?